can/message.py

```diff
@@ -99,7 +100,13 @@ def __init__(self, timestamp=0.0, arbitration_id=0, extended_id=True,
 
         self.timestamp = timestamp
         self.arbitration_id = arbitration_id
-        self.is_extended_id = extended_id
+        if is_extended_id is not None:
+            self.is_extended_id = is_extended_id
+        else:
+            if not extended_id:
+                # Passed extended_id=False (default argument is True) so we warn to update
+                warnings.warn("extended_id is a deprecated parameter, use is_extended_id", DeprecationWarning)
```

Comment:
A little late but I don't really understand this. We only want to warn when `extended_id=False` is used but not `extended_id=True`?

And I think we should wait with issuing a warning until one or two versions later. Otherwise it won't be possible to support more than the latest python-can version without having thousands of warnings in the log.

Reply:
Or we could print a single warning per interpreter "session".